### ISSUE TITLE
CSS interpolate-size property and calc-size() function

### ISSUE THREAD

**Issue by dbaron** [mozilla/standards-positions#1022]
## Request for Mozilla Position on an Emerging Web Specification

* Specification title: CSS calc-size() function
* Specification or proposal URL (if available): https://drafts.csswg.org/css-values-5/#calc-size
* Explainer URL (if available): https://github.com/w3c/csswg-drafts/blob/main/css-values-5/calc-size-explainer.md
* Proposal author(s): @tabatkins @dbaron @bfgeek
* Caniuse.com URL (optional): none
* Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=1896734
* Mozillians who can provide input (optional): @emilio @dholbert
* WebKit standards-position: https://github.com/WebKit/standards-positions/issues/348

**Comment by dbaron** [comment]
I've updated the title to also include the `interpolate-size` property, particularly given the rationale in https://github.com/w3c/csswg-drafts/issues/10294#issuecomment-2127202025 .

**Comment by tats-u** [comment]
@dholbert Why are you putting off the decision so long? There should be no privacy invasion factor in this feature.

**Comment by dholbert** [comment]
Yeah, sorry; this seems fine from a standards-positions perspective; it's a reasonable approach to address a long-standing problem in the web platform.

(I've just fallen behind on my standards-positions backlog and need to catch up at some point.)

**Comment by dholbert** [comment]
@zcorpan I added `position:positive`; let me know if we need anything else here or if this can be closed. Sorry again for the delay.

**Comment by tats-u** [comment]
@dholbert Thank you for your quick response. I am glad to get the "positive" answer from you. I hope that this will encourage the implementation of this feature.

**Comment by tedw** [comment]
FYI there’s a caniuse.com URL now https://caniuse.com/mdn-css_types_calc-size
